File: Hari01/Dart/kalkulator.dart
double kalkulator(double a, double b, String operator) {
  switch (operator) {
    case '+':
      return a + b;
    case '-':
      return a - b;
    case '*':
      return a * b;
    case '/':
      return a / b;
    default:
      print("Operator tidak valid");
      return 0;
  }
}

void main() {
  print(kalkulator(10, 5, '+')); // Output: 15.0
}

File: Hari01/Pascal/kalkulator.pas
program Kalkulator;
var
  a, b: real;
  operator: char;
begin
  writeln('Masukkan angka pertama:');
  readln(a);
  writeln('Masukkan operator (+, -, *, /):');
  readln(operator);
  writeln('Masukkan angka kedua:');
  readln(b);
  
  case operator of
    '+': writeln('Hasil: ', a + b:0:2);
    '-': writeln('Hasil: ', a - b:0:2);
    '*': writeln('Hasil: ', a * b:0:2);
    '/': writeln('Hasil: ', a / b:0:2);
  else
    writeln('Operator tidak valid');
  end;
end.
